docker/Makefile

```diff
@@ -0,0 +1,66 @@
+git_describe := $(shell git describe)
+version := $(shell echo $(git_describe) | sed 's/-.*//')
```

Comment:
I'm not sure if this Makefile would also target Windows, but we won't have access to things like `sed`, or `/dev/null` (Windows has `NUL` which is similar there, but nothing as full-featured as `sed`)

Reply:
@Iristyle I thought make didn't really work on windows?

Reply:
It's more that it's not a default tool and fairly uncommon (except to maybe C++ folks). CMake would be the more "Microsoft approved" way of doing things, and even then, you at least have to install a choco package to get going.

I wasn't sure if the intent was to use `make` everywhere, but if this is only for non-Windows (where make is far more common), then the Windows concerns aren't that important.